packages/tool_util/test-requirements.txt

```
@@ -1,3 +1,3 @@
-beaker
+beaker==1.11.0 ; python_version >= "3.7" and python_version < "3.12"
```

```suggestion
Beaker
```

The Beaker pin in `pyproject.toml` seems unnecessary, I'll open a separate PR to verify that we can remove it from there.

---

https://github.com/galaxyproject/galaxy/pull/15665